src/h5web/vis-packs/core/line/LineVis.tsx

```diff
@@ -71,9 +65,8 @@ function LineVis(props: Props) {
     );
   }
 
-  const [abscissas, abscissaToIndex] = useMemo(() => {
-    const abscissas = abscissaValue || range(dataArray.size);
-    return [abscissas, getValueToIndexScale(abscissas, true)];
+  const abscissas = useMemo(() => {
+    return abscissaValue || range(dataArray.size);
```

Comment:
`return` could be omitted to have a one-liner perhaps ?

Reply:
Unfortunately no, the dependency array gets split onto a couple of lines 😢